From e1abc8aae14a4c82295b442a4553f18dae24774f Mon Sep 17 00:00:00 2001
From: Snejp <bartosz.dudziak@snejp.pl>
Date: Thu, 4 Jun 2020 12:32:29 +0200
Subject: [PATCH 1/5] remove "Unified Sysfs for GPU" kernel option

This line was added with Samsung Galaxy S5(SM-G900H) kernel import and is not needed.
---
 drivers/gpu/Makefile | 1 -
 1 file changed, 1 deletion(-)

diff --git a/drivers/gpu/Makefile b/drivers/gpu/Makefile
index c9355dfc..01cef64b 100644
--- a/drivers/gpu/Makefile
+++ b/drivers/gpu/Makefile
@@ -1,3 +1,2 @@
 obj-y			+= drm/ vga/ stub/ ion/
 obj-$(CONFIG_MSM_KGSL)	+= msm/
-obj-y += gpu_sysfs/
-- 
2.25.1

